src/status_im/async_storage/core.cljs

```diff
@@ -0,0 +1,46 @@
+(ns status-im.async-storage.core
+  (:require [re-frame.core :as re-frame]
+            [taoensso.timbre :as log]
+            [goog.functions :as f]
+            [status-im.async-storage.transit :refer [clj->transit transit->clj]]
+            ["@react-native-community/async-storage" :default async-storage]))
+
+(def ^:private debounce-ms 250)
+
+(def key->string str)
+
+(defn- set-item! [key value]
+  (-> ^js async-storage
+      (.setItem (key->string key)
```

Comment:
so these keys will be shared between multiaccounts ?

Reply:
if you hide banner it won't be visible in the new multiaccount right ?

Reply:
Yes, it is on the app level